src/database/entities/user.entity.ts

```diff
@@ -0,0 +1,106 @@
+import { Entity, PrimaryGeneratedColumn, Column } from 'typeorm';
+
+@Entity('auth.users')
+export class UserEntity {
+  @PrimaryGeneratedColumn('uuid')
+  readonly id: string;
+
+  @Column({ nullable: true })
+  instance_id: string;
+
+  @Column()
+  aud: string;
+
+  @Column({ nullable: true })
+  role: string;
+
+  @Column()
+  email: string;
+
+  @Column()
+  encrypted_password: string;
+
+  @Column({ nullable: true })
+  email_confirmed_at: Date;
+
+  @Column({ nullable: true })
```

Comment:
Tem muitas colunas nullable aqui, e são muitas propriedades, você acha que faria sentido dividir essa entidade em outras entidades de forma que a gente tenha numa tabela os dados primários comuns, e em outra tabela ou tabelas os dados que podem ser preenchidos depois?

Ou ao menos adicionar presunções para dados que são escolhas binárias.

Tipo `is_super_admin`, se ele não for true, necessariamente pode se considerar false, so há duas opções, não precisa deixar `nullable: true`

Reply:
apenas respeitando a estrutura da tabela, mas por logica defalt false para esse caso seria o correto, mas se for falar em correto, essa estrutura da users parece que foi feito em um curso da udemy